tests/test_E901.py

```diff
@@ -23,7 +23,7 @@ def lasting(self, duration=300):
 '''
         errors = errors_from_src(src)
         if sys.version_info < (3, 12):  # pragma: <3.12 cover
-            expected = ['E122:4:1', 'E251:5:13', 'E251:5:15']
+            expected = ['E122:4:1']
```

Comment:
these were due to the E251 check under flowing (which it no longer does now that we track parens in a stack)